### DISCUSSION TITLE
HTML Documentation

### DISCUSSION THREAD

**Discussion by kamiliahs** [codeigniter4/shield#373] [Q&A] [answered]
There is some documentation in html such as main codeigniter project ?

**Comment by datamweb** [comment]
Hello, in **HTML** form, no.
But you can see our documents [here](https://github.com/codeigniter4/shield/blob/develop/docs/index.md).

**Comment by kenjis** [comment, marked as answer]
See https://codeigniter4.github.io/shield/

**Reply by MGatner** [reply]
Beautiful!
